docs/get_double.adoc

```diff
@@ -11,28 +11,46 @@ get_double - prompts user for a line of text from stdin and returns the equivale
 
 *#include <cs50.h>*
 
-*double get_double(string prompt);*
+*double get_double(const string format, ...);*
 
 == DESCRIPTION
 
-Prompts user for a line of text from standard input and returns the equivalent double as precisely as possible; if text does not represent a double or if value would cause underflow or overflow, user is prompted to retry.
+Prompts user for a line of text from standard input and returns the equivalent double as precisely as possible; if text does not represent a double or would cause underflow or overflow, user is reprompted.
+
+The prompt is formatted like *printf(3)*.
 
 == RETURN VALUE
 
-Returns the equivalent double of the line read from stdin, as precisely as possible. If line can't be read, returns *DBL_MAX*.
+Returns the double equivalent to the line read from stdin in [*DBL_MIN*, *DBL_MAX*), as precisely as possible. If line can't be read, returns *DBL_MAX*.
 
 == EXAMPLE
 
-    // attempt to read double from stdin
-    double d = get_double("Enter double: ");
+....
+// Returns the quotient of two doubles, or DBL_MAX on error
```

Comment:
For comments atop functions, we've historically used `/** ... */`, though we could perhaps switch to `//` for those too. But either way, let's have comments atop functions be full sentences, just like in `cs50.h`.